kasa/smartprotocolerrors.py

```diff
@@ -0,0 +1,86 @@
+"""Module for defining SMART protocol errors."""
+from enum import Enum
+
+
+class ErrorCode(Enum):
+    """Enum for SMART Error Codes."""
+
+    SUCCESS = 0
+
+    UNKNOWN_ERROR_CODE = 999999
+    ERROR_CODE_NONE = 999998
+    ERROR_CODE_INVALID = 999997
```

Comment:
Please order and group these into logical groups based on the first digit(s) of the error code, that'll make it much easier to check them out.

Reply:
Done